modin/pandas/accessor.py

```diff
@@ -191,3 +195,72 @@ def __get__(self, obj, cls):
         accessor_obj = self._accessor(obj)
         object.__setattr__(obj, self._name, accessor_obj)
         return accessor_obj
+
+
+class ExperimentalFunctions:
+    """
+    Namespace class for accessing experimental functionality.
+
+    Parameters
+    ----------
+    data : DataFrame or Series
+        Object to operate on.
+    """
+
+    def __init__(self, data):
+        if IsExperimental.get() is not True:
```

Comment:
Can we get rid of this requirement? I mean, if a user already gone that far by accessing an underscore `df._exp` field, then he probably understands what he's doing and his intention to use experimental API is certain enough.

Reply:
we could have added a decorator for each of the methods in `ExperimentalFunctions` that would enable/disable experimental mode in the fly

Reply:
@dchigarev done, ready for review